src/zip/zip.go

```diff
@@ -18,6 +18,12 @@ type ZipHandler struct {
 	zipClient        ZipClient
 }
 
+type ZipHandlerInterface interface {
```

Comment:
Is ZipClient a better name?

Reply:
tragically that name is already taken (at the bottom of this file). I'm open to suggestions on renaming either/both

Reply:
I like putting the word Interface or perhaps an I at the end of the name to mark it as an interface